packages/components/src/__actions__/Button/v3/Button.tsx

```diff
@@ -45,11 +51,12 @@ export const Button = forwardRef(
       isPending,
       hasHiddenPendingLabel: propsHasHiddenPendingLabel = false,
       pendingLabel,
+      isReversed = false,
       ...restProps
     }: ButtonProps,
     ref: React.ForwardedRef<HTMLButtonElement>
   ) => {
-    const isReversed = useReversedColors()
+    const isReversedVariant = useReversedColors() || isReversed
```

Comment:
I think hooks need to be called as they are.
Also, the prop should override the context imo, given it's explicit whereas the context is more magical.